src/runmode-af-xdp.c

```diff
@@ -335,6 +394,37 @@ int RunModeIdsAFXDPSingle(void)
 
     (void)ConfGet("af-xdp.live-interface", &live_dev);
 
+    const char *live_dev_backup = NULL;
+    if (live_dev == NULL) {
+        live_dev_backup = LiveGetDeviceName(0);
+        live_dev = live_dev_backup;
+    }
+
+    // get af_xdp_node from config
+    ConfNode *af_xdp_node = NULL;
+    af_xdp_node = ConfGetNode("af-xdp");
+    if (af_xdp_node == NULL)
+        SCLogInfo("unable to find af-xdp config using default values");
+
+    // from the af_xdp node get the config root for an interface and default
+    ConfNode *if_root = NULL;
+    ConfNode *if_default = NULL;
+    if_root = ConfFindDeviceConfig(af_xdp_node, live_dev);
+    if_default = ConfFindDeviceConfig(af_xdp_node, "default");
+    if (if_root == NULL && if_default == NULL) {
+        SCLogInfo("unable to find af-xdp config for "
+                  "interface \"%s\" or \"default\", using default values",
+                live_dev);
+    }
+
+    // get the path to the xdp-program
+    const char *program_path = NULL;
+    const char *confstr = NULL;
+    if (ConfGetChildValueWithDefault(if_root, if_default, "xdp-program", &confstr) == 1) {
```

Comment:
I think I'm missing something. Shouldn't `xdp-program` be added to our `suricata.yaml.in` if you want to use that config value?